department-of-veterans-affairs/vets-website
# MHV-55332 MHV-58203 Update-Content-And-Left-Navigation-For-Labs-Chem/Hem

## Summary
This ticket combines the tasks from MHV-55332-Update-Content-And-Left-Navigation-For-Labs-Chem and MHV-58203-Change-word-on-all-print-download. After discussing with UCD, I added "Lab and test results" information to the left side of the navigation bar on the landing page. Additionally, on the VitalDetail page, the download drop-down list items were rephrased by replacing "page" with "list."

Note: It was decided that in ticket MHV-55332, the addition of the settings button to the left navigation will be handled in a new ticket.

## Related issue(s)

[MHV-55332](https://jira.devops.va.gov/browse/MHV-55332) - Update Content and Left Navigation for Labs Chem/Hem
[MHV-58203](https://jira.devops.va.gov/browse/MHV-58203)- Change the wording on all print/download buttons on vitals lists

MHV-55332 - Update Content and Left Navigation for Health Conditions-see latest designs 

https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=4467-36464&t=NouvL14iCrDlRZ5S-0 

From Amelia:  We will need to include the settings page and download all records page. Content for those is most likely changing, too–hopefully we will have those change asap.  (Do we need a separate ticket?)

User Story: As a Medical Records User, I want to see updated content and left navigation so that the latest designs are displayed.

Feature Flag Y/N ? N

Manual Testing Y/N ? Y

Automated Testing Y/N ? N

Accessibility Testing Y/N ? N

UCD Validation Y/N Y

Acceptance Criteria:

AC1  Left Navigation has been updated to latest design

AC2 Content has been updated to latest design

AC3 Verify with UCD 
AC4
AC5

Links to UCD:

Figma Link: https://www.figma.com/design/mGZRdLypKGaFsHo5xp2kaZ/Medical-Records?node-id=4467-36464&t=NouvL14iCrDlRZ5S-0

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

MHV-58203 - Change the wording on all print/download buttons on vitals lists to say "Print this list" and "download a PD of this list" and "Download a text file (.txt) of this list".

From Amelia:  

User Story:  As a MR user, I want to 

Feature Flag Y/N ? N

DataDog/Analytics?  Y/N ?

Manual Testing Y/N ? Y-Maruf

Automated Testing Y/N ?

Accessibility Testing Y/N ? Y-Bobby

UCD Validation Y/N Y

Acceptance Criteria:
AC1  
AC2  
AC3 UCD has validated changes (Amelia)
AC4

Links to UCD:
Figma Link:   
Other Design Notes

Architectural Notes:

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots

![Screenshot 2024-05-20 at 4 29 37 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/ce3fba28-8219-46af-b453-469c57cb4834)
![Screenshot 2024-05-20 at 4 30 05 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/9de54642-a743-4786-b7b7-4fe48dbdb9c2)
![Screenshot 2024-05-21 at 1 38 01 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/2b761b89-151b-4909-8228-e6194f2dd7e1)



## What areas of the site does it impact?

The modification affects the Landings page and VitalDetail page of the Medical Records pages.

## Acceptance criteria
"Lab and test results" information to the left side of the navigation bar on the landing page
VitalDetail page, the download drop-down list items were rephrased by replacing "page" with "list."

### Quality Assurance & Testing


### Authentication
